spec/index.bs

```diff
@@ -1358,7 +1361,7 @@ interface GPUAdapter {
 <script type=idl>
 dictionary GPUDeviceDescriptor : GPUObjectDescriptorBase {
     sequence<GPUFeatureName> features = [];
-    GPULimits limits = {};
+    record<DOMString, GPUSize32> limits = {};
```

Comment:
not all limits will be u32 though. Maybe this should be Number?

Reply:
As mentioned in the comment, we can change this later by changing it to `double` or `any` and writing out the type conversions explicitly for each limit. It's easier to have this now because we don't have to do that.